app/javascript/styles/_resultsPanel.scss

```diff
@@ -126,3 +126,18 @@
     margin: 0 0.2em 0 0.2em;
   }
 }
+
+.study-type {
+  border: 1px solid #4d72aa;
+  background-color: white;
+  font-weight: normal;
+  color: #777;
+  width: 44px;
+  height: 20px
+}
+
+.study-label {
+  grid-template-columns: 6fr 0.25fr;
```

Comment:
This rule is causing issues with the inferred search "text match" label that shows up when we run a search using a facet, but then also attempt to make a text match using the facet label.  Example: running a search on production using `disease: COVID-19` will match some studies because they have "COVID-19" in their title or description.  This displays a badge like this with a tooltip explaining what the match means:

![Screen Shot 2021-06-29 at 11 28 46 AM](https://user-images.githubusercontent.com/729968/123826215-8e2ad880-d8cd-11eb-849a-139f94352d1e.png)

This new CSS rule will result in those badges looking like this:

![Screen Shot 2021-06-29 at 11 28 59 AM](https://user-images.githubusercontent.com/729968/123826296-a4389900-d8cd-11eb-84d6-33bfd9c4a36e.png)

I think we either need a different class for this particular badge, or we need to handle the grid spacing or word wrapping differently.

Reply:
I think this is fixed, I also moved the "HCA" badge over so it's not far-right-aligned anymore so didn't change the coloring, please take a look and let me know though if you still think that would be helpful!